docs/contributing/pages/components.mdx

```diff
@@ -193,6 +192,7 @@ Attributes:
 
 - `header` (string) - optional header value to include, rendered as an H2
 - `nextPages` (boolean) - only render pages which come next based on sidebar ordering
+- `exclude` (string[]) - an array of pages to exclude from the grid. Specify the file name of the page, for example, `"index"` for `index.mdx`.
```

Comment:
hmm looks like this file wasn't formatted previously. I only added this line, rest of the file is just auto-format.